conf/AppsConfig.go

```diff
@@ -92,3 +100,15 @@ func GetIosAppsViaConfig() AllIosApps {
 
 	return iosApps
 }
+
+func GetSchedulerConfigsViaConfig() AllSchedulerConfigs {
```

Comment:
The method name `GetSchedulerConfigsViaConfig` looks confusing
I think we can just have `GetSchedulerConfigs` as the method name

Reply:
Done Prabhat